src/functionalTest/java/uk/gov/hmcts/darts/configuration/AzureAdB2CAuthenticationProperties.java

```diff
@@ -17,5 +17,7 @@ public class AzureAdB2CAuthenticationProperties {
     private String password;
     private String clientId;
     private String clientSecret;
+    private String globalUsername;
```

Comment:
Do we need these 2 properties here - or have they been replaced by `AzureAdB2CGlobalAuthenticationProperties` ?